test/verify/machineslib.py

```diff
@@ -1619,6 +1677,9 @@ def installWithErrorTest(dialog):
         self.allow_journal_messages('.*Connection.*')
         self.allow_journal_messages('.*session closed.*')
 
+        self.allow_browser_errors("Failed when connecting: Connection closed")
```

Comment:
Since these are errors because of disconnecting the console as well iit would make sense that you would put them grouped with the other line 2 lines bellow.
But i don't mind merging like this of course.